discover/index.html

```diff
@@ -64,9 +64,12 @@ <h3 class="text-left" id="connect"><i class="fab fa-fw fa-mixcloud"></i> Get Con
 <p>If you are looking for <strong>non-licensed imagery downloads</strong> (NAIP, HRO, etc), check out the <a href="{% link data/aerial-photography/index.html %}">aerial photography</a> section of the data pages.</p>
 <h3 class="text-left" id="services"><i class="far fa-fw fa-map"></i> Available Services</h3>
 <ul class="dotless no-padding">
-  <li><i class="fas fa-fw fa-lock"></i> <code>Utah</code> – Combination of the 2018 NAIP and latest Google imagery with scale-dependent rendering <strong>(licensed)</strong>
-  <li><i class="fas fa-fw fa-lock"></i> <code>Hexagon 15cm</code> – 6-inch natural-color <a href="{% link data/aerial-photography/index.html %}#HEXAGON">aerial photography from Hexagon</a> collected November 2019 covering the Wasatch Front, Logan, and St. George<strong> (licensed)</strong>
-  <li><i class="fas fa-fw fa-lock"></i> <code>Hexagon 30cm</code> – 12-inch statewide natural-color <a href="{% link data/aerial-photography/index.html %}#HEXAGON">aerial photography from Hexagon</a> collected in 2018<strong> (licensed)</strong>
+  <li><i class="fas fa-fw fa-lock"></i> <code>Utah</code> – Combination of the 2018 Hexagon 30cm imagery and the 2019 Hexagon 15cm imagery with scale-dependent rendering <strong>(licensed)</strong>
+    <ul class="dotless no-margin">
+      <li><em>This layer now uses the Hexagon imagery. The rural areas have a slightly lower resolution than the Google imagery but are more up-to-date. The most recent Google imagery is still available in the <code>Google</code> service.</em>
```

Comment:
**suggestion**: clarify what was replaced
```suggestion
      <li><em>This layer now uses the Hexagon imagery instead of the Google imagery. The rural areas have a slightly lower resolution than the Google imagery but are more up-to-date. The most recent Google imagery is still available in the <code>Google</code> service.</em>
```

Reply:
Good fix; I'm doing a separate commit to remove the number of "Google imagery"s in that block.